Source/JavaScriptCore/Configurations/DebugRelease.xcconfig

```diff
@@ -37,6 +37,8 @@ CODE_SIGN_IDENTITY_YES = $(WK_ENGINEERING_CODE_SIGN_IDENTITY);
 SDKROOT = $(SDKROOT_$(USE_INTERNAL_SDK));
 SDKROOT_ = macosx;
 SDKROOT_YES = macosx.internal;
+ADDITIONAL_SDKS = $(ADDITIONAL_SDKS_$(USE_INTERNAL_SDK));
```

Comment:
Any chance that we'll ever want `$(inherit)` here? Probably best to add it everywhere preemptively if so.

Reply:
Good idea. I'd like to extend this mechanism for downlevels, at which point we'd need a declaration in Base.xcconfig.